### PR TITLE
Use Rebalance function in test

### PR DESCRIPTION
**Pull request by cbrit**

### CODE REVIEW

File: integration_tests/cork_test.go [hunk 1]
```diff
@@ -152,8 +169,12 @@ func (s *IntegrationTestSuite) TestCork() {
 		aave_abi, err := AaveV2MetaData.GetAbi()
 		s.Require().NoError(err)
 
-		methodName := "claimAndUnstake"
-		method, err := aave_abi.Pack(methodName)
+		methodName := "rebalance"
+		route := [9]common.Address{common.HexToAddress(dai), common.HexToAddress(pool), common.HexToAddress(usdc), common.HexToAddress(zeroAddr), common.HexToAddress(zeroAddr), common.HexToAddress(zeroAddr), common.HexToAddress(zeroAddr), common.HexToAddress(zeroAddr), common.HexToAddress(zeroAddr)}
```
Comment (EricBolten) [inline integration_tests/cork_test.go]: You don't really need to call `comon.HexToAddress(zeroAddr)` over and over again, you could just assign it as a variable.

File: integration_tests/cork_test.go [hunk 2]
```diff
@@ -272,7 +293,15 @@ func (s *IntegrationTestSuite) TestCork() {
...
 			if len(logs) == 1 {
 				s.T().Log("saw mock function event!")
-				return true
+
+				log := logs[0]
+				if len(log.Data) > 0 {
+					var event AaveV2MockRebalance
+					err := aave_abi.UnpackIntoInterface(&event, "mockRebalance", log.Data)
+					s.Require().NoError(err, "failed to unpack mockRebalance event from log data")
+					s.Require().Equal(event.MinAssetsOut, big.NewInt(1000))
```
Comment (EricBolten) [inline integration_tests/cork_test.go:303]: If you're going to inspect `MinAssetsOut`, is it worth also checking the route and swap values?

**Review by EricBolten — COMMENTED** [review]
Some comments.

File: integration_tests/cork_test.go (hunk 2 above)
Comment (cbrit) [reply integration_tests/cork_test.go:303]: Seemed unnecessarily complicated to me?

File: integration_tests/cork_test.go (hunk 1 above)
Comment (cbrit) [reply integration_tests/cork_test.go]: ah meant to fix this

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**